commercial-paper/organization/digibank/application-java/pom.xml

```diff
@@ -14,7 +14,7 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
     <!-- fabric-chaincode-java -->
-    <fabric-chaincode-java.version>[2.0,3.0)</fabric-chaincode-java.version>
+    <fabric-chaincode-java.version>2.1.0</fabric-chaincode-java.version>
```

Comment:
I don't liking pinning to what will become an outdated java chaincode version, but if it is required, let's add a comment explaining the motivation for future generations.

Reply:
Something actually broke in the dependency chain. You actually cant compile this example anymore, using any version other than `2.1.0` has dependency conflicts, several people have noted this in RocketChat this past week. I need to open a Jira to have the Java team investigate this

Reply:
Ok, let's go ahead and merge. FYI @lesleyannjordan @bestbeforetoday this could use some investigation.